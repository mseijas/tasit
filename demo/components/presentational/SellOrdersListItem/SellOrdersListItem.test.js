import React from "react";
import { shallow } from "enzyme";
import SellOrdersListItem from "./SellOrdersListItem";

describe("SellOrdersListItem", () => {
  it("renders the component", async () => {
    expect(
      shallow(<SellOrdersListItem sellOrder={{}} onPress={() => {}} />)
    ).toMatchSnapshot();
  });
});
