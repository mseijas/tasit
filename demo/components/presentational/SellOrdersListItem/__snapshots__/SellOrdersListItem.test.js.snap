// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellOrdersListItem renders the component 1`] = `
<TouchableHighlight
  activeOpacity={0.85}
  delayPressOut={100}
  onPress={[Function]}
  underlayColor="black"
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginBottom": 13.34,
        "paddingBottom": 40.019999999999996,
        "paddingLeft": 22.5,
        "paddingRight": 22.5,
        "paddingTop": 40.019999999999996,
      }
    }
  >
    <SellOrder
      sellOrder={Object {}}
    />
  </Component>
</TouchableHighlight>
`;
